### PR TITLE
tests: add additional test cases

### PR DESCRIPTION
**Pull request by adriantam**
## Description
Add additional test cases for testing userset and TTU.  In additional, add error traces in some error cases. 

## Review Checklist
- [X] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [ ] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [X] The correct base branch is being used, if not `main`
- [X] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

File: assets/tests/consolidated_1_1_tests.yaml [hunk 1]
```diff
@@ -8407,4 +8554,789 @@ tests:
...
+        listUsersAssertions:
+          - request:
+              filters:
+                - user
+              object: document:1
+              relation: viewer
+            expectation:
+              - user:anne
+              - user:bob
+      # notice we are removing group2
```
Comment (willvedd) [inline assets/tests/consolidated_1_1_tests.yaml:9024]: **Suggestion: ** Relocate this to line 9307

**Review by willvedd — COMMENTED** [review]
Another test scenario I'm not seeing is multiple non-userset type restrictions in tupleset relation pointing to a relation with multiple non-userset type restrictions.

```
				model
					schema 1.1
				type user1
				type user2
				type group1
					relations
						define member: [user1, user2]
				type group2
					relations
						define member: [user1, user2]
				type folder
					relations
						define owner: [group1, group2]
						define viewer: member from owner
```
And then a permutation of that where one of the type restrictions doesn't have the `member` relation:

```
```
				model
					schema 1.1
				type user1
				type user2
				type group1
					relations
						define member: [user1, user2]
				type folder
					relations
						define owner: [group1, group2]
						define viewer: member from owner
```

File: assets/tests/consolidated_1_1_tests.yaml (hunk 1 above)
Comment (adriantam) [reply assets/tests/consolidated_1_1_tests.yaml:9024]: 9307 is a different test (ttu_orphan_public_wildcard_parent)

Comment (adriantam) [reply assets/tests/consolidated_1_1_tests.yaml:9024]: I think you meant 9037

**Comment by adriantam** [comment]
> relations_not_defined_in_some_child_type_falsy

added test ttu_some_parent_type_removed

**Comment by willvedd** [comment]
>[Test relations_not_defined_in_some_child_type_falsy](https://github.com/openfga/openfga/blob/ca319c24d797a4e5cc00142f934d04cd23be87ac/assets/tests/consolidated_1_1_tests.yaml#L4266) has something very similar. The only difference is that nothing is assigned to group2 member from the beginning.

It's similar but different because the computed relation from the TTU only has one type restriction associated. I think it warrants a test because this situation is seemingly novel and want to make sure that the fast path is handled correctly here. However, I can add that in later with #1765.

**Review by willvedd — DISMISSED** [review]

File: assets/tests/consolidated_1_1_tests.yaml [hunk 2]
```diff
@@ -8407,4 +8629,790 @@ tests:
...
+          type group1
+            relations
+              define member: [user, user:*]
+          type group2
+            relations
+              define member: [user, user:*]
+          type document
+            relations
+              define parent: [group1, group2]
+              define viewer: member from parent
```
Comment (miparnisari) [inline assets/tests/consolidated_1_1_tests.yaml:9052]: this is the same model as the test preceding it, can we collapse both tests into one? in other words: the assertions on this test, are they testing something new compared to the preceding test?

Comment (adriantam) [reply assets/tests/consolidated_1_1_tests.yaml]: combined test into one

**Review by willvedd — APPROVED** [review]

**Pull request merged**